utils/update_nginx_conf_dev.sh

```diff
@@ -1,15 +0,0 @@
-#!/bin/sh
```

Comment:
why delete this file? no relate with healthcheck

Reply:
we do not need this file, we use openresty `luajit` always for now.